### PR TITLE
Fix styling of arguments

### PR DESCRIPTION
**Pull request by ashlymat**
Signed-off-by: Ashly Mathew <ashlymathew93@gmail.com>

**- What I did**
Small contribution to #923 - changing arguments of options to italics

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by ashlymat** [comment]
@kolyshkin Could you do a review?

**Comment by thaJeztah** [comment]
Looks like CI got stuck (probably because some changes were made); let me try a close/reopen to trigger it

**Review by thaJeztah — APPROVED** [review]
changes LGTM, thanks!

File: man/dockerd.8.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ $ sudo dockerd --add-runtime runc=runc --add-runtime custom=/usr/local/bin/my-ru
   Set parent cgroup for all containers. Default is "/docker" for fs cgroup
   driver and "system.slice" for systemd cgroup driver.
 
-**--config-file**="/etc/docker/daemon.json"
+**--config-file**=*"/etc/docker/daemon.json"*
```
Comment (kolyshkin) [inline man/dockerd.8.md:154]: 1. This is wrong to put a default value (like done here) or an example value in synopsis. The proper way would be to say something like

```markdown
**--config-file**=*"path"*
```

and add
```markdown
Default is  */etc/docker/daemon.json*.
```
to the option description.

Comment (kolyshkin) [reply man/dockerd.8.md:154]: To explain the last part -- while a file name (such as `/etc/docker/daemon.json`) is a literal, the man page convention is to typeset it as italic.

File: man/dockerd.8.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ $ sudo dockerd --add-runtime runc=runc --add-runtime custom=/usr/local/bin/my-ru
 **-D**, **--debug**=*true*|*false*
   Enable debug mode. Default is false.
 
-**--default-cgroupns-mode**="**host**|**private**"
+**--default-cgroupns-mode**="*host*|*private*"
```
Comment (kolyshkin) [inline man/dockerd.8.md:168]: Here, both `host` and `private` are literals, i.e. they should be used as-is. Therefore, those are typeset in bold and should not be changed.

OTOH, if that would be something like `default-cgroup-mode=MODE`, the MODE should be in italic, as it's a value to be substituted.

**Comment by kolyshkin** [comment]
Opened https://github.com/docker/cli/pull/4215 to fix the issues from this PR (described above).

**Pull request merged**